cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1582,3 +1582,20 @@ if(webgui)
      INSTALL_COMMAND ""
      SOURCE_DIR ${CMAKE_BINARY_DIR}/ui5/distribution)
 endif()
+
+#------------------------------------------------------------------------------------
+# Look for libatomic. On ARM systems we need to explicitly link against that library
+# in certain places. But if it's not available, then assume that we are on a system
+# on which it is not needed.
+#
+string(REPLACE ":" ";" LD_LIBRARY_PATH "$ENV{LD_LIBRARY_PATH}")
```

Comment:
Since this is later unset (unconditionally), consider using a more 'unique' name?

Reply:
Yeah, I was wondering about that myself as well. Okay, I'll name this something less generic.

Reply:
I looked around the code a bit for some convention to follow. Since in some places I saw that you guys use `_` prefixed variables exactly like we do in the ATLAS CMake code as well, I went with such a naming. Hopefully that's along the lines that you were thinking of Philippe. :wink:

Reply:
As it turns out my code was pretty bad there. :frowning: One should really not do replacements like this. All "find commands" of CMake can be told to consider environment variables with the `ENV` keyword. So now I updated the code to do just that.